Makefile

```diff
@@ -127,8 +127,26 @@ coverage:
 clean:
 	-rm -rf ${CURRENT_DIR}/dist
 
+$(GOPATH)/bin/go-to-protobuf:
```

Comment:
This doesn't work when the project is cloned out of $GOPATH, please use the way like

https://github.com/argoproj/argo-events/blob/master/hack/update-codegen.sh